tractor/_testing/pytest.py

```diff
@@ -111,3 +128,164 @@ async def _main():
         return trio.run(main)
 
     return wrapper
+
```

Comment:
Aforementioned `pytest` plugin factoring for usage in other `tractor` dependents using the [pointer mentioned in the descr](https://github.com/goodboy/tractor/pull/380#issue-3221311792).